src/fsharp/FSharp.Core/control.fs

```diff
@@ -1201,7 +1184,7 @@ namespace Microsoft.FSharp.Control
                 args.aux.trampolineHolder.Protect((fun () ->
                     if completedTask.IsCanceled then
                         if useCcontForTaskCancellation
-                        then args.aux.ccont (new OperationCanceledException(args.aux.token))
```

Comment:
Hmm... I thought this change was done in a separate PR (or we decided not to do it in a separate PR....)